integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -226,18 +226,15 @@ public static void initAll(@Namespaces(6) List<String> namespaces) {
   @Tag("gate")
   @Tag("crio")
   void testK8SEventsSuccess() {
-    try {
-      OffsetDateTime timestamp = now();
-      logger.info("Creating domain");
-      createDomain(domainNamespace1, domainUid, pvName1, pvcName1);
+    OffsetDateTime timestamp = now();
```

Comment:
Why remove the finally block, here? I thought we kept those to prevent a failure in one test from breaking others.

Reply:
The shutdown of the domains at the end of a test makes it hard to see the domain's status at the failure. We need to make the test cases independent from each other in a different way, IMO.

Reply:
In addition, the test cases that have been changed in this PR are testing domains in different namespaces; in theory, these test cases should not impact each other.

Reply:
@sankarpn can you comment on this? thanks.

Reply:
I am ok with removing the finally clause.